Casks/m/magicavoxel.rb

```diff
@@ -1,15 +1,33 @@
 cask "magicavoxel" do
-  version "0.99.6.2"
+  version "0.99.6.2,0.99.6"
   sha256 "4ee661e41da07f8c2b2b1c916bd1e18f7c5229900f30467eeea73ad74e861226"
 
-  url "https://github.com/ephtracy/ephtracy.github.io/releases/download/#{version.major_minor_patch}/MagicaVoxel-#{version}-macos-10.15.zip",
+  url "https://github.com/ephtracy/ephtracy.github.io/releases/download/#{version.csv.second || version.csv.first}/MagicaVoxel-#{version.csv.first}-macos-10.15.zip",
       verified: "github.com/ephtracy/ephtracy.github.io/"
   name "MagicaVoxel"
   desc "8-bit 3D voxel editor and interactive path tracing renderer"
   homepage "https://ephtracy.github.io/"
 
+  # Upstream doesn't provide a macOS file with every release, so we have to
+  # check multiple GitHub releases instead of only the "latest" one.
   livecheck do
-    skip "No reliable way to get version info"
+    url :url
+    regex(/MagicaVoxel[._-]v?(\d+(?:\.\d+)+)[._-]macos[._-]10.15\.zip/i)
```

Comment:
```suggestion
    regex(/MagicaVoxel[._-]v?(\d+(?:\.\d+)+)[._-]macos.*?\.zip/i)
```

There may be something to be said for glossing over the macOS version in this context. If we check for a specific macOS version, it could quietly omit a newer version that uses a different macOS version in the filename. If we gloss over the macOS version (and don't include it in the cask `version`), we would need to manually update the `url` but at least livecheck would correctly inform us that a new version is available.